src/public/client.ts

```diff
@@ -41,16 +41,40 @@ class DisplayMap {
         <small class="form-text text-muted" style="display: inline-block;">PORT: ${data.port}</small>
         <hr />
         <div class="deleteButton" href="/">Delete</div>
+        <div class="deleteButton edit" href="/" style="padding: 0px 0px 0px 20px;">Edit</div>
     </li>
 `
+
+      const delButton = helper.getElement('.deleteButton', mappingElement)
+      delButton.onclick = (): void => {
+        fetch(`/api/mappings/delete/${data.id}`, {
+          method: 'DELETE',
+          body: JSON.stringify({ data }),
+          headers: {
+            'Content-Type': 'application/json'
+          }
+        }).then(()=>{
+          window.location.reload()
+        })
+      }
+      const editButton = helper.getElement('.edit', mappingElement)
```

Comment:
could you add a comment below this line? `//TODO: turn into input boxes for ability to edit.`